tools/create-translation-issues.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+# This script opens translation issues for all languages via the Github API.
+# The script is idempotent and can be run multiple times. It will not create
+# multiple issues if milestone and title remain the same. Instead, it will
+# update any existing issues with a (potentially) updated body text.
+#
+# Syntax:
+#   ./tools/create-translation-issues.sh RELEASE DEADLINE app|web [EXTRA_TEXT]
+#
+# Requirements:
+# - Github CLI tools + valid login
+# - jamulus and jamulussoftware git checkouts
+#
+# Example usage for app translations:
+#   ./tools/create-translation-issues.sh 3.8.0 2021-05-15 app 'Note: The term "Central server" has been replaced with "Directory server"'
+# Example usage for web translations:
+#  ./jamulus/tools/create-translation-issues.sh 3.8.0 2021-05-15 web 'Note: The term "Central server" has been replaced with "Directory server"'
```

Comment:
Should be `../` at the front of the example.

Reply:
Yep, thanks, updated.